src/Compilers/Test/Utilities/VisualBasic/CompilationTestUtils.vb

```diff
@@ -923,18 +926,18 @@ Friend Module CompilationUtils
         Dim expectedReader = New StringReader(expected)
         Dim actualReader = New StringReader(actual)
 
-        Dim expectedBuilder As New StringBuilder()
-        Dim actualBuilder As New StringBuilder()
+        Dim expectedBuilder = _PooledStringBuilderPool.Allocate()
```

Comment:
> Doesn't really matter in vb.net 

Yes, compiler will eat it, but out coding style requires the capitalization of references to match declaration.